program/program/Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Text;
+
+namespace  ConsoleApp
+{
+    class Program
+    {
+        static void Main()
+        {
+
+            //First task
+            Console.WriteLine("First task:");
+            string stringCompare1 = "Hello";
+            string stringCompare2 = "Hello";
+            string sortABC = "Hello";
+            string dublicate = "Hello and hi";
+
+
+
+
+            bool compare = stringCompare1==stringCompare2;
+            Console.WriteLine($"Is string equal: {compare}");
+
+            //Second task
+            Console.WriteLine("\nSecond task:");
+
+            string example1 = "afASDsd12345";                  
+            
+
+            Console.Write("numbers = ");
+            Console.WriteLine(CountOfNumbers(example1));
+            Console.Write("letter = ");
+            Console.WriteLine(CountOfLetters(example1));
+
+            //Third task
+            Console.WriteLine(SortByAlphabetical(sortABC));
+
+            //Fourth task
+
+            Console.WriteLine(DublicateFilter(dublicate));
+
+        }
+
+        static int CountOfLetters(String example1)
+        {
+            int letter = 0;
+
+            for (int i = 0; i < example1.Length; i++)
+            {
+                if ((example1[i] >= 'A' && example1[i] <= 'Z') || (example1[i] >= 'a' && example1[i] <= 'z'))
```

Comment:
it can be done via char.IsLetter method that check not only for English alphabet